src/resources/api_nw_newwin.js

```diff
@@ -369,15 +369,23 @@ NWWindow.prototype.toggleKioskMode = function() {
   currentNWWindowInternal.toggleKioskModeInternal(this.cWindow.id);
 };
 
-NWWindow.prototype.showDevTools = function(frm, callback) {
-  var id = '';
-  if (typeof frm === 'string')
-    id = frm;
-  var f = null;
+/**
+ * Open the devtools to inspect the new window.
+ * 
+ * @param {string | HTMLIFrameElement} iframe
+ * @param {Function} callback
+ */
+NWWindow.prototype.showDevTools = function(iframe, callback) {
+  if (process.versions['nw-flavor'] !== 'sdk')
+    throw new Error('showDevTools does not exist on normal build flavor.');
```

Comment:
If you aren't in the SDK, running this command should probably just do nothing and then return. I think the error is fine, as it at least conveys you are in Normal NW.js, which can happen unexpectedly, as mentioned in the issue via `npm install` of `^0.75.0-sdk` (where the `^` will look for `0.75.x` instead of the actual `sdk` version you wanted, this is an annoying quirk of how npm works).
```suggestion
  if (process.versions['nw-flavor'] !== 'sdk') {
    throw new Error('showDevTools does not exist on normal build flavor.');
    return;
  }
```